hw02_unpack_string/go.mod

```
@@ -1,11 +1,11 @@
-module github.com/fixme_my_friend/hw02_unpack_string
+module unpack
```

Следует указать полный путь до модуля
```suggestion
module github.com/diletech/hw-test/hw02_unpack_string
```